ksql-engine/src/main/java/io/confluent/ksql/function/udaf/max/DoubleMaxKudaf.java

```diff
@@ -29,7 +29,7 @@
 public class DoubleMaxKudaf extends KsqlAggregateFunction<Double, Double> {
 
   DoubleMaxKudaf(int argIndexInValue) {
-    super(argIndexInValue, () -> Double.MIN_VALUE, Schema.FLOAT64_SCHEMA,
+    super(argIndexInValue, () -> -Double.MAX_VALUE, Schema.FLOAT64_SCHEMA,
```

Comment:
what happens if we encounter Double.NEGATIVE_INFINITY?

Reply:
Good point - thanks @xvrl will use that instead

Reply:
you may want to look at the implications at serialization time though. For instance, there is no standard representation in JSON for infinities. Jackson stores them as strings by default. As long as you stick to jackson you should be fine, but other systems may break.